html/semantics/interactive-elements/the-dialog-element/dialog-requestclose.html

```diff
@@ -35,14 +35,8 @@
   assert_true(dialog.matches(':open'));
   assert_equals(dialog.matches(':modal'),openMethod === 'modal');
 }
-function getSignal(t) {
-  const controller = new AbortController();
-  const signal = controller.signal;
-  t.add_cleanup(() => controller.abort());
-  return signal;
-}
```

Comment:
As a small aside, this `getSignal` function is effectively `t.get_signal()`, so I refactored this also.